server/service/src/invoice_line/inbound_shipment_service_line/delete/validate.rs

```diff
@@ -19,9 +17,9 @@ pub fn validate(
     connection: &StorageConnection,
 ) -> Result<InvoiceLineRow, DeleteInboundShipmentServiceLineError> {
     let line = check_line_exists(&input.id, connection)?;
-    let invoice = check_invoice_exists(&input.invoice_id, connection)?;
+    let invoice = check_invoice_exists(&line.invoice_id, connection)?;
 
-    check_line_belongs_to_invoice(&line, &invoice)?;
+    // check_line_belongs_to_invoice(&line, &invoice)?;
```

Comment:
Remove...

I might miss something but has check_line_belongs_to_invoice been removed, is it still needed?